### ISSUE TITLE
Crosshair disappears

### ISSUE THREAD

**Issue by Gireen** [Unvanquished/Unvanquished#1450]
Happened for the first time that the crosshair just disappeared. 
Changing settings, weapons and teams had no effect.
Affected where all crosshairs for aliens and humans.

**Comment by necessarily-equal** [comment]
Maybe can you describe what you were doing or otherwise give more information? Any idea how to reproduce?

**Comment by Gireen** [comment]
No. It was the first time that this happened to me and so far also not again, it might be some rare problem.
